test/util/zip.js

```diff
@@ -8,12 +8,14 @@ const streamTest = require('streamtest').v2;
 // calls the callback with an object as follows:
 // {
 //      filenames: [ names of files in zip ],
-//      {filename}: "contents",
-//      {filename}: "contents",
-//      …
+//      files: {
```

Comment:
Might be worth mentioning that `files` is a `Map`?

Reply:
Or could we remove the whole comment?  The structure can be seen on line 18, and can also be understood from existing examples and at runtime by inspection.

Reply:
That makes sense. :+1: